app/assets/javascripts/modules/remove-filter.js

```diff
@@ -36,7 +35,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
       if (inputType === 'checkbox') {
         $input.prop('checked', false)
-        $input.trigger(onChangeSuppressAnalytics)
+        window.GOVUK.triggerEvent($input[0], 'change', suppressAnalytics)
```

Comment:
If you're extracting element `[0]` of `$input`, that means that `$input` is still a jQuery object. You're going to need to update line 68 inside the `getInput` function.

Reply:
On the same subject, line 19 is also creating a jQuery object: `var $el = $(e.target)`

Reply:
And lines 21-24 need to change from `data` to `getAttribute('data-`

Reply:
And lines 32-34... `prop` is jQuery, and so is `val()`

Reply:
Line 61 also creates a jQuery object - `$('#' + removeFilterFacet)`

Reply:
So i might be misunderstanding things here. Do these bits need to be done for this PR? I was only trying to remove jQuery from the expander. The changes to `remove-filter` were needed to get it working because I'd changed an event listener to be JS and it couldn't hear jQuery triggers. These comments seem to be related to removing jQuery from the `remove-filter` file which isn't something I was trying to do in this PR. Planning on doing that later

Reply:
Oh I see... sorry. Got confused. That's fine then, although if you're already doing part of it and you're going to do it anyway, I'd go for including it in this PR as well, but up to you.

Reply:
yeah, i think I'd prefer to merge as is just to keep each PR as nuclear as possible